### PR TITLE
chore: change windows and linux command for redo

### PR DESCRIPTION
**Pull request by MayurSMahajan**
I have changed the default shortcut for redo operation on Windows and Linux from `ctrl+shift+z` to `ctrl+y`.
Partially Solves: [#2888](https://github.com/AppFlowy-IO/AppFlowy/issues/2888)

### CODE REVIEW

**Comment by LucasXu0** [comment]
I suggest keeping both commands. Is it common to use Ctrl + Y in both Linux and Windows?

**Comment by MayurSMahajan** [comment]
In windows and linux it is common to use ctrl+y, but ctrl+shift+z is also supported. Okay lets support both

File: test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+import 'package:flutter_test/flutter_test.dart';
+
+import '../../../infra/testable_editor.dart';
+import '../../../util/util.dart';
+
+// single | means the cursor
+// double | means the selection
+void main() async {
+  setUpAll(() {
+    if (kDebugMode) {
```
Comment (Xazin) [inline test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart:15]: Do you need this check?

File: test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart [hunk 2]
```diff
@@ -0,0 +1,152 @@
...
+      key: LogicalKeyboardKey.keyZ,
+      isMetaPressed: true,
+      isShiftPressed: true,
+    );
+  } else {
+    await editor.pressKey(
+      key: LogicalKeyboardKey.keyY,
+      isControlPressed: true,
+    );
+  }
```
Comment (Xazin) [inline test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart:140]:
```suggestion
  await editor.pressKey(
    key: Platform.isMacOS ? LogicalKeyboardKey.keyZ : LogicalKeyboardKey.keyY,
    isMetaPressed: Platform.isMacOS,
    isShiftPressed: Platform.isMacOS,
    isControlPressed: !Platform.isMacOS,
  );
```

File: test/service/internal_key_event_handlers/redo_undo_handler_test.dart [hunk 3]
```diff
@@ -269,24 +237,38 @@ Future<void> _testBackspaceUndoRedo(
...
+      key: LogicalKeyboardKey.keyZ,
+      isMetaPressed: true,
+      isShiftPressed: true,
+    );
+  } else {
+    await editor.pressKey(
+      key: LogicalKeyboardKey.keyY,
+      isControlPressed: true,
+    );
+  }
```
Comment (Xazin) [inline test/service/internal_key_event_handlers/redo_undo_handler_test.dart:273]:
```suggestion
  await editor.pressKey(
    key: Platform.isMacOS ? LogicalKeyboardKey.keyZ : LogicalKeyboardKey.keyY,
    isMetaPressed: Platform.isMacOS,
    isShiftPressed: Platform.isMacOS,
    isControlPressed: !Platform.isMacOS,
  );
```

File: test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart (hunk 1 above)
Comment (MayurSMahajan) [reply test/new/service/shortcuts/command_shortcut_events/undo_redo_command_test.dart:15]: Yes, it activates logging when during debugging the tests

**Pull request merged**